packages/core/src/browser/keybinding.spec.ts

```diff
@@ -14,9 +14,15 @@
 // SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 // *****************************************************************************
 
-import { enableJSDOM } from '../browser/test/jsdom';
+import { enableJSDOM } from './test/jsdom';
 let disableJSDOM = enableJSDOM();
 
+import { FrontendApplicationConfigProvider } from './frontend-application-config-provider';
+import { ApplicationProps } from '@theia/application-package/lib/application-props';
+FrontendApplicationConfigProvider.set({
+    ...ApplicationProps.DEFAULT.frontend.config
+});
```

Comment:
Suggestion: Would it make sense to extract this into a dedicated method? I'm open for the current style, but it's just a lot of redundancy.

Reply:
@msujew I cleaned up the code, and existing usages of `FrontendApplicationConfigProvider.set` since it already handles populating the default configuration values. Is it the idea you had in mind?